install/databases.js

```diff
@@ -69,11 +76,19 @@ function saveDatabaseConfig(config, databaseConfig, callback) {
 			password: databaseConfig['mongo:password'],
 			database: databaseConfig['mongo:database'],
 		};
+	} else if (config.database === 'postgres') {
+		config.postgres = {
+			host: databaseConfig['postgres:host'],
+			port: databaseConfig['postgres:port'],
+			username: databaseConfig['postgres:username'],
+			password: databaseConfig['postgres:password'],
+			database: databaseConfig['postgres:database'],
+		};
 	} else {
 		return callback(new Error('unknown database : ' + config.database));
 	}
 
-	var allQuestions = questions.redis.concat(questions.mongo);
+	var allQuestions = questions.redis.concat(questions.mongo).concat(questions.postgres);
```

Comment:
`git grep ')\.concat(' src` returns quite a few places where the `.concat().concat()` style is used.